CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ Here is an overview of all new **experimental** features:
 
 - **General**: Use (self-signed) certificates for all the communications (internals and externals) ([#3931](https://github.com/kedacore/keda/issues/3931))
 - **Redis Scalers**: Add support to Redis 7 ([#4052](https://github.com/kedacore/keda/issues/4052))
+- **RabbitMQ Scaler**:  Add TLS support ([#967](https://github.com/kedacore/keda/issues/967))
```

Comment:
Sorry for being a pain, could you move this before Redis? we try to keep the changelog alphabetically sorted
https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog

Reply:
Hey, no issues, I will change it.